AppCenterCrashes/AppCenterCrashes/MSCrashes.mm

```diff
@@ -127,6 +133,14 @@ @interface MSCrashes ()
  */
 @property BOOL didCrashInLastSession;
 
+/**
+ * Indicates that the app received a low memory warning in the last session.
+ * It is possible that a low memory warning was sent but couldn't be logged if iOS killed the app before updating the flag in
+ * the filesystem. Apps can also be killed without receiving a low memory warning, or receive the warning, but crash for another reason.
+ *
+ * @warning This property only has a correct value, once the sdk has been properly initialized!
```

Comment:
It's also not guaranteed to be correct because of the comment above, right?  Does this sound more correct?

```suggestion
 * @warning This property only has an updated value once the SDK has been properly initialized!
```